freqtrade/persistence/trade_model.py

```diff
@@ -1502,7 +1502,7 @@ class Trade(ModelBase, LocalTrade):
     id: Mapped[int] = mapped_column(Integer, primary_key=True)  # type: ignore
 
     orders: Mapped[List[Order]] = relationship(
-        "Order", order_by="Order.id", cascade="all, delete-orphan", lazy="selectin",
+        "Order", order_by="Order.id", cascade="all, delete-orphan", lazy="selection",
```

Comment:
Careful review, please.

Reply:
Yeah, this one explicitly refers to `selectin` in the sqlalchemy "world" - selection is not correct here, and makes essentially every test fail (at least the ones that touch this file)